admin/inc/functions_themes.php

```diff
@@ -6,9 +6,6 @@
  * Website: http://www.mybb.com
  * License: http://www.mybb.com/about/license
  *
- */
```

Comment:
Why are you changing this?

Reply:
Why not? Looks better IMHO

Reply:
Both comments serve entirely different purposes, so it makes very little sense to merge them.